[Workflows] Core components - triggers (#4696)
elastic/docs-content: explore-analyze/workflows/triggers/manual-triggers.md

---

# Manual triggers

Manual triggers run workflows on-demand through the UI or API. They require explicit user action to start a workflow. Use manual triggers for testing, one-off tasks, administrative actions, or workflows that require a human decision to start.

> The following example shows the basic syntax for a manual trigger:

Reviewer: ```suggestion
To define a manual trigger, use the following syntax:
```

My idea here is to make it more to-the-point (simpler).
